pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,12 +54,13 @@ type Response struct {
 
 type Client struct {
 	*client.Client
-	Request  Request
-	Response Response
-	Error    bool
-	Created  bool
-	server   *httptest.Server
-	handler  *utiltesting.FakeHandler
+	Clientset *release_1_1.Clientset
```

Comment:
Release 1.1 clientset is not locked to a version yet. Its inputs and outputs are unversioned API objects, but its Codec can be versioned, and its url path can be versioned. So maybe we don't need the comment.